docs/guide/logger.rst

```diff
@@ -46,15 +46,17 @@ or also by::
 You can use it to issue your own messages or to change its defaults. If you'd like to
 capture messages as they are generated you can do that with a context manager::
 
-    with log.log_to_list() as log_list:
-        # your code here
+    >>> from sunpy import log
+    >>> with log.log_to_list() as log_list:
```

Comment:
This needs a `  #doctest: +SKIP` on the end or it will fail doctests and the line below should be `...` not `>>>`. ( and will also need a skip)

Reply:
done